classes/script_metadata.php

```diff
@@ -83,6 +83,32 @@ class script_metadata {
     /** Default sample limit. */
     const SAMPLE_LIMIT_DEFAULT = 1024;
 
+    /** @var int Stack limit config. */
+    protected static $stacklimit;
+    /** @var int Sample limit config. */
+    protected static $samplelimit;
+    /** @var int Sampling period */
+    public static $samplems;
+    /** @var string */
+    protected static $redactparams;
+
+    /**
+     * Preload config values to avoid DB access during processing.
```

Comment:
maybe add another sentence around why this is so important

Reply:
There is already some documentation on this, but I will expand on it.